pkg/maps/maps.go

```diff
@@ -0,0 +1,24 @@
+package maps
+
+import (
+	"fmt"
+	"strings"
+)
+
+func Join[M ~map[K]V, K comparable, V any](m M, sliceSep, mapSep string) string {
```

Comment:
Another go-commons candidate.

Needs a comment explaining what it does.

Reply:
Added